app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx

```diff
@@ -1,120 +1,140 @@
-import { ticketsList } from "connectors";
+import { useState } from "react";
 import TicketListPage from "./Page";
 import { FormattedMessage as T } from "react-intl";
-import { substruct } from "fp";
+import { useTicketsList } from "../hooks";
+import * as txTypes from "constants/Decrediton";
 
 const labels = {
-  unknown: <T id="ticket.status.multiple.unknown" m="unknown" />,
-  unmined: <T id="ticket.status.multiple.unmined" m="unmined" />,
-  immature: <T id="ticket.status.multiple.immature" m="immature" />,
-  live: <T id="ticket.status.multiple.live" m="live" />,
-  voted: <T id="ticket.status.multiple.voted" m="voted" />,
-  missed: <T id="ticket.status.multiple.missed" m="missed" />,
-  expired: <T id="ticket.status.multiple.expired" m="expired" />,
-  revoked: <T id="ticket.status.multiple.revoked" m="revoked" />
+  [txTypes.UNKNOWN]: <T id="ticket.status.multiple.unknown" m="unknown" />,
```

Comment:
Thanks for that!

Reply:
Indeed thanks 🔥 

Kill all these TODOs 🔪

Reply:
@victorgcramos